sw/device/lib/dif/dif_otbn.h

```diff
@@ -0,0 +1,167 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_DIF_DIF_OTBN_H_
+#define OPENTITAN_SW_DEVICE_LIB_DIF_DIF_OTBN_H_
+
+#include "sw/device/lib/base/mmio.h"
+#include <stddef.h>
+#include <stdint.h>
+
+// Header Extern Guard (so header can be used from C and C++)
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+/**
+ * Width of wide instructions in OTBN, in bit.
+ */
+#define OTBN_WLEN 256
```

Comment:
This needs to be prefixed like other DIF symbols: `DIF_OTBN_WLEN`. Also, I would consider including the units in the name, e.g. `DIF_OTBN_WLEN_BITS`, since I think most readers of this value elsewhere might expect it to be a length in bytes.

Consider making this a constant instead.